commit da71990c74342f5370bf54408f86f1bf5d1bf2b9
Author: sunnavy <sunnavy@bestpractical.com>
Date:   Mon Aug 22 05:40:59 2011 +0800

    show same page menu in simple search as in normal search if there is query

diff --git a/share/html/Elements/Tabs b/share/html/Elements/Tabs
index ff2055b..f4e8ce91 100755
--- a/share/html/Elements/Tabs
+++ b/share/html/Elements/Tabs
@@ -571,7 +571,7 @@ my $build_admin_menu = sub {
             }
         }
     }
-    if ( $request_path =~ m{^/(?:Ticket|Search)/} && $request_path !~ m{^/Search/Simple\.html}) {
+    if ( $request_path =~ m{^/(?!Search/Simple\.html)(?:Ticket|Search)/} || $request_path =~ m{^/Search/Simple\.html} && $m->request_args->{'q'} ) {
         my $search = Menu()->child('search');
         my $args      = '';
         my $has_query = '';
